source/Frame.lua

```diff
@@ -78,12 +78,15 @@ local function Frame(name, parent)
         return false
     end
 
+    ---@class Frame
```

Comment:
added `@class` annotation to frame object for emmy lua based language server users _e.g., sumneko, IntelliJ Emmy Lua_